### ISSUE TITLE
[terra-theme-provider] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-framework#1176]
# Bug Report

The following themeable variables are missing:

**clinical-lowlight-theme**

--terra-theme-provider-mock-background-color - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 3, column: 4)
--terra-theme-provider-mock-color - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 4, column: 4)
--terra-theme-provider-mock-display - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 5, column: 4)

**orion-fusion-theme**

--terra-theme-provider-mock-background-color - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 3, column: 4)
--terra-theme-provider-mock-color - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 4, column: 4)
--terra-theme-provider-mock-display - packages/terra-theme-provider/lib/terra-dev-site/doc/example/MockThemeComponent.module.scss (line: 5, column: 4)
